src/OrbitGl/PagefaultTrack.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "PagefaultTrack.h"
+
+#include "Geometry.h"
+#include "TextRenderer.h"
+#include "TimeGraph.h"
+#include "TimeGraphLayout.h"
+#include "Viewport.h"
+
+namespace orbit_gl {
+
+PagefaultTrack::PagefaultTrack(CaptureViewElement* parent, TimeGraph* time_graph,
```

Comment:
I think in the future, we should try to make the tracks more flexible, and in this case here have more shared code between the GpuTrack and this track.

Reply:
Make sense. We can have a class of the composite track and move the shared code of managing subtracks to the new class.

Reply:
Agree. Let's keep this in our minds.